content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,206 @@ aliases:
+
+| Metric Name       | Tag Values                           | Aggregations                            |
+| ----------------- | ------------------------------------ | --------------------------------------- |
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+
+##### Distribution Metrics with Additional Percentile Aggregations
+
+[Adding additional percentile aggregations][2]: `p50`, `p75`, `p90`, `p95`, `p99`) to your DISTRIBUTION metric increases the number of distinct custom metrics by raising the ammount of aggregations. The final number of metric is:
+
+```
+Number_of_distinct_tag_combinations * ( 5 + Number_of_additional_percentile_aggregations_choosen)
+```
+
+[1]: /developers/metrics/types/?tab=distribution
+[2]: /graphing/metrics/distributions/#aggregations
+{{% /tab %}}
+{{< /tabs >}}
+
+### Effect of Adding Tags
+
+Adding tags **may not** result in more custom metrics. Your count of custom metrics usually scales with the most granular/detailed tag. Let’s suppose you’re measuring temperature in the US and you’ve tagged your temperature metric by country and region. You submit the following data to Datadog:
+
+| Metric Name   | Tag Values                         |
+| ------------- | ---------------------------------- |
+| `temperature` | `country:USA`, `region: Northeast` |
+| `temperature` | `country:USA`, `region: Southeast` |
+
+Suppose you wanted to add the tag `City` which has three values: `NYC`, `Miami`, and `Orlando`. Adding this tag does increase the number of custom metrics since it provides more detail and granularity to your dataset as shown below:
+
+| Metric Name   | Tag Values                                          |
+| ------------- | --------------------------------------------------- |
+| `temperature` | `country:USA`, `region: Northeast`, `city: NYC`     |
+| `temperature` | `country:USA`, `region: Southeast`, `city: Orlando` |
+| `temperature` | `country:USA`, `region: Southeast`, `city: Miami`   |
+
+The count of custom metrics reporting from `temperature` scales with the most granular tag, `city`.
+
+Now suppose you wanted to tag your temperature metric also by `state` (which has 2 values: `NY` and `Florida`). You’re now tagging temperature by the set of tags: `country`, `region`, `state`, and `city`. Adding the state tag doesn’t increase the level of granularity already present in your dataset provided by the city tag.
```

Comment:
"Now suppose you also wanted to tag your temperature metric by `state` (which has two values: `NY` and `Florida`)."